### DISCUSSION TITLE
How to spoof browser info when using is_mobile?

### DISCUSSION THREAD

**Discussion by Vrackko** [seleniumbase/SeleniumBase#2754] [Q&A] [answered]
When i am using android user agent and test my bot on pixelscan, it detects that android device is using chrome. Is there way to spoof that. I am already using is_mobile but obviously that doesn't do the trick.

![image](https://github.com/seleniumbase/SeleniumBase/assets/145341123/0c3c77dc-c03d-4e81-b610-f666b1f2be07)

**Comment by mdmintz** [comment, marked as answer]
What's the issue with using Chrome on a mobile device running Android? Android devices use Chrome.

The Selenium mobile emulator is very detectable, and there's no real way to hide that you're using an emulator because a real mobile device works differently.

If you're trying to get past Pixelscan, just use a regular UC Mode web browser:

```python
from seleniumbase import SB

with SB(uc=True, incognito=True, test=True) as sb:
    sb.driver.uc_open_with_reconnect("https://pixelscan.net/", 10)
    sb.remove_elements("jdiv")  # Remove chat widgets
    sb.assert_text("No automation framework detected", "pxlscn-bot-detection")
    not_masking = "You are not masking your fingerprint"
    sb.assert_text(not_masking, "pxlscn-fingerprint-masking")
    sb.highlight("span.text-success", loops=8)
    sb.sleep(1)
    sb.highlight("pxlscn-fingerprint-masking div", loops=9, scroll=False)
    sb.sleep(1)
    sb.highlight("div.bot-detection-context", loops=10, scroll=False)
    sb.sleep(2)
```

Or the simple `Driver()` version of that script:

```python
from seleniumbase import Driver

driver = Driver(uc=True)
driver.uc_open_with_reconnect("https://pixelscan.net", "breakpoint")
driver.quit()
```

<img width="1350" alt="Screenshot 2024-01-02 at 10 32 53 AM" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/2dd0ed1f-7fae-47ce-9c19-6e2c757069fb">
